packages/react/src/PasswordInput/PasswordInput.test.tsx

```diff
@@ -0,0 +1,84 @@
+import { render, screen } from '@testing-library/react'
+import { createRef } from 'react'
+import { PasswordInput } from './PasswordInput'
+import { Label } from '../Label'
+import '@testing-library/jest-dom'
+
+describe('Password input', () => {
+  it('renders', () => {
+    const { container } = render(<PasswordInput />)
+
+    const component = container.querySelector(':only-child')
+
+    expect(component).toBeInTheDocument()
+    expect(component).toBeVisible()
+  })
+
+  it('renders a design system BEM class name', () => {
+    const { container } = render(<PasswordInput />)
+
+    const component = container.querySelector(':only-child')
+
+    expect(component).toHaveClass('ams-password-input')
+  })
+
+  it('renders an additional class name', () => {
+    const { container } = render(<PasswordInput className="extra" />)
+
+    const component = container.querySelector(':only-child')
+
+    expect(component).toHaveClass('ams-password-input extra')
+  })
+
+  it('supports ForwardRef in React', () => {
+    const ref = createRef<HTMLInputElement>()
+
+    const { container } = render(<PasswordInput ref={ref} />)
+
+    const component = container.querySelector(':only-child')
+
+    expect(ref.current).toBe(component)
+  })
+
```

Comment:
Did you remove the following tests from TextInput on purpose? They seem relevant to me.

- should be working in a controlled state
- should not update the value when disabled

Reply:
No, not sure where I copied the tests from. Added now.